server/.gitignore

```diff
@@ -1,3 +1,107 @@
-node_modules
+# Logs
```

Comment:
where did all this come from?

Reply:
I used [gitignore.io](gitignore.io) which generates it from open-source templates

Reply:
I used [gitignore.io](gitignore.io) which generates it from open-source templates